GithubCallbackTrigger/__init__.py

```diff
@@ -43,42 +44,48 @@ def get_details(pr):
     if "dependabot" in pr.user.login and pr.user.type == "Bot":
         _, package, _, _, _, version_to = pr.title.split(" ", 6)
 
-    return package, version_to
+    if package:
+        package = package.lower()
 
+    return package, version_to
 
-def main(req: func.HttpRequest) -> func.HttpResponse:
-    if "X-Hub-Signature-256" not in req.headers:
-        return func.HttpResponse(f"Go away", status_code=403)
 
-    digest = (
+def make_digest(body:bytes) -> str:
```

Comment:
I moved this into a function so that I could call it from tests. The tests were failing without it.